docs/client-sdk/javascript/tutorials/start-messaging.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Start messaging with XMTP
+sidebar_position: 1
+---
+
+# Start messaging with XMTP
+
+XMTP is an open protocol and network for secure web3 messaging. 
+
+Developers build with [XMTP client SDKs](/sdks-and-tools) to provide messaging in their apps. The XMTP messaging API client takes care of:
+
+- [Authentication](/docs/dev-concepts/account-signatures) using an **XMTP identity that you own and control**
+- [End-to-end encryption](/docs/dev-concepts/invitation-and-message-encryption) of **messages that you own and control**
+- Providing an **[interoperable inbox](/docs/dev-concepts/interoperable-inbox)** accessible across apps built with XMTP
+- Relaying messages to the **progressively decentralized** [XMTP network](/docs/dev-concepts/architectural-overview#network-layer)
+
+A great way to start learning how to build an app with XMTP is to use one!
+
+**To start messaging with XMTP:**
+
+1. **Pick an identity** you want to use to message with XMTP.
```

Comment:
The only 'identity' we support today is EVM wallets. ENS & Lens handles are popular resolvers (they point to an EVM address). Can we use language that clarifies that? 

E.g. today people can message my wallet using yashluna.eth. However, if I change the redirection (yashluna.eth -> 0xabcd...1234 instead of yashluna.eth -> 0xqwer....6789), people are messaging a different _identity_ in a way when they message yashluna.eth

Reply:
YES - thank you for this, @yash-luna - such a great call out! Updating now.